shared/utils/src/test/roundvalue.unit.spec.js

```diff
@@ -13,47 +13,68 @@ tape('roundValue tests', function (test) {
 	let value = 1.23456789e-10
 	let rounded = 1.2e-10
 	test.equal(roundValueAuto(value), rounded, `should return ${rounded}`)
+	test.equal(roundValue2(value), value, `is unchanged`)
 
 	value = 1.2345
 	rounded = 1.23
 	test.equal(roundValueAuto(value), rounded, `should return ${rounded}`)
-
-	value = 0.000056
-	rounded = 5.6e-5
-	test.equal(roundValueAuto(value), rounded, `should return ${rounded}`)
+	test.equal(roundValue2(value), rounded, `should return ${rounded}`)
 
 	value = 10549.23556789
 	rounded = '1.1e+4'
+	let r2 = 10549
+	test.equal(roundValueAuto(value, true), rounded, `should return ${rounded}`)
```

Comment:
what does `roundValueAuto(value)` return? add test

Reply:
In that instance it's probably best to use `roundValue` with the zero digits. There's nothing inherently wrong with these functions. Rather when to use them is, at least to me, opaque. It would be helpful to create a guideline of how numbers ought to be rendered. For instance scientific notation should be used for axis and scales, but not for user inputs and raw counts. Maybe even a blurb in roundValue.js would be sufficient.

Reply:
this is the usecase that roundValueAuto() doesn't handle well (two decimals unnecessary), and it's not possible to use roundValue(x,0) since that violates generalization policy. still the issue of two decimals are minor and okay to merge

<img width="252" alt="image" src="https://github.com/user-attachments/assets/8af66de2-1348-4565-811b-2d700d2c3e4c">